<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Edit_PoruchStatus</class>
 <widget class="QDialog" name="Edit_PoruchStatus">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>597</width>
    <height>438</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Назначение поручения</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="17" column="0" colspan="2">
       <widget class="QPushButton" name="pushButton_prinyat">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Принять</string>
        </property>
        <property name="icon">
         <iconset resource="../resourses.qrc">
          <normaloff>:/img/img/535786.png</normaloff>:/img/img/535786.png</iconset>
        </property>
       </widget>
      </item>
      <item row="9" column="4">
       <widget class="QPushButton" name="pushButton_openFile">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Открыть файл</string>
        </property>
        <property name="icon">
         <iconset resource="../resourses.qrc">
          <normaloff>:/img/img/1134892.png</normaloff>:/img/img/1134892.png</iconset>
        </property>
       </widget>
      </item>
      <item row="17" column="2">
       <widget class="QPushButton" name="pushButton_otkl">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Отклонить</string>
        </property>
        <property name="icon">
         <iconset resource="../resourses.qrc">
          <normaloff>:/img/img/3996.png</normaloff>:/img/img/3996.png</iconset>
        </property>
       </widget>
      </item>
      <item row="14" column="0" colspan="5">
       <widget class="QDateEdit" name="dateEdit">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="displayFormat">
         <string>dd MMMM yyyy</string>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="8" column="0" colspan="5">
       <widget class="QLabel" name="label_5">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Файл</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0" colspan="5">
       <widget class="QPlainTextEdit" name="plainTextEdit_text">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item row="13" column="0" colspan="5">
       <widget class="QLabel" name="label_6">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Дата сдачи</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0" colspan="5">
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Описание</string>
        </property>
       </widget>
      </item>
      <item row="17" column="4">
       <widget class="QPushButton" name="pushButton_cancel">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Отмена</string>
        </property>
        <property name="icon">
         <iconset resource="../resourses.qrc">
          <normaloff>:/img/img/cancel.png</normaloff>:/img/img/cancel.png</iconset>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="5">
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Заголовок поручения</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="5">
       <widget class="QLineEdit" name="lineEdit_title">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Исполнитель:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="4">
       <widget class="QLabel" name="label_ispolnitel">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Исполнитель</string>
        </property>
       </widget>
      </item>
      <item row="17" column="3">
       <widget class="QPushButton" name="pushButton_wait">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>В ожидании проверки</string>
        </property>
        <property name="icon">
         <iconset resource="../resourses.qrc">
          <normaloff>:/img/img/loader2.gif</normaloff>:/img/img/loader2.gif</iconset>
        </property>
       </widget>
      </item>
      <item row="9" column="0" colspan="4">
       <widget class="QLineEdit" name="lineEdit_file">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEdit_title</tabstop>
  <tabstop>plainTextEdit_text</tabstop>
  <tabstop>lineEdit_file</tabstop>
  <tabstop>dateEdit</tabstop>
  <tabstop>pushButton_cancel</tabstop>
 </tabstops>
 <resources>
  <include location="../resourses.qrc"/>
 </resources>
 <connections/>
</ui>
